### PR TITLE
Fix Group Performance

### PR DESCRIPTION
**Pull request by yotamberk**
Fixed the bug that I caused...

### CODE REVIEW

**Comment by yotamberk** [comment]
@wimrijnders I found what caused my stupid mistake... Please review this

**Comment by yotamberk** [comment]
@wimrijnders Of course!
The problem was in `_updateItemsVerticalPosition` returning `false` or a `resized`. The condition was wrong and shouldn't have been in there.

**Comment by yotamberk** [comment]
@wimrijnders can you approve the merge again? I had to add another commit and it cancelled the review

File: lib/timeline/component/Group.js [hunk 1]
```diff
@@ -285,20 +285,14 @@ Group.prototype._applyGroupHeight = function(height) {
 }
 
 // update vertical position of items after they are re-stacked and the height of the group is calculated
-Group.prototype._updateItemsVerticalPosition = function(resized, margin) {
+Group.prototype._updateItemsVerticalPosition = function(margin) {
```
Comment (wimrijnders) [inline lib/timeline/component/Group.js]: Damn, I should have noticed that one, even if timeline is not my expertise.

File: lib/timeline/component/Group.js [hunk 2]
```diff
@@ -360,8 +354,15 @@ Group.prototype.redraw = function(range, margin, forceRestack, returnQueue) {
...
     // update vertical position of items after they are re-stacked and the height of the group is calculated
     (function() {
-      return resized = this._updateItemsVerticalPosition.bind(this)(resized, margin)
-    }).bind(this)
+      this._updateItemsVerticalPosition.bind(this)(resized, margin)
+    }).bind(this),
+
+    function() {
+      if (!this.isVisible && this.height) {
+        resized = false;
```
Comment (wimrijnders) [inline lib/timeline/component/Group.js]: But `resized` is a boolean, right? I still don't understand why previous went wrong, but it really isn't important.

**Pull request merged**